### PR TITLE
ml4ht integration

### PR DESCRIPTION
**Pull request by ndiamant**
Adds the ability to use `DataDescription` from https://github.com/broadinstitute/torch_ml4h as `TensorMap` in the model factory, and to use `TensorMap`s as `DataDescription`s in ml4ht data loaders.

### CODE REVIEW

File: ml4h/ml4ht_integration/tensor_map.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+import h5py
+
+from ml4h.TensorMap import TensorMap, Interpretation
+from ml4ht.data.data_description import DataDescription
+from ml4ht.data.defines import SampleID, LoadingOption, Tensor, Batch
+
+
+class TensorMapSampleGetter:
+    def __init__(
```
Comment (mklarqvist) [inline ml4h/ml4ht_integration/tensor_map.py:11]: We need a comment here with what this is and how to use it (including an example)

File: ml4h/ml4ht_integration/tensor_map.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+
+def tensor_map_from_data_description(
+        data_description: DataDescription,
+        interpretation: Interpretation,
+        shape: Tuple[int, ...],
+        **tensor_map_kwargs,
+) -> TensorMap:
+    """
+    Allows a DataDescription to be used in the model factory
```
Comment (mklarqvist) [inline ml4h/ml4ht_integration/tensor_map.py:51]: We need to expand on this comment, including describing the parameters, an example, and examples of valid kwargs.

**Review by mklarqvist — CHANGES_REQUESTED** [review]
Needs only improved documentation otherwise good. 👍

**Review by mklarqvist — APPROVED** [review]
👍

**Review by lucidtronix — APPROVED** [review]
LGTM!

**Comment by lucidtronix** [comment]
I think this all got merged via #447,  @ndiamant feel free to reopen if I'm mistaken.

**Comment by mklarqvist** [comment]
🎉  @ndiamant

**Pull request closed without merging**